docs/source/atomclient.rst

```diff
@@ -0,0 +1,104 @@
+.. _atom-client-module:
+
+==================
+Atom Client Module
+==================
+
+Description
+===========
+
+This module will periodically fetch from one or more `Atom feed <https://en.wikipedia.org/wiki/Atom_%28standard%29>`_ sources and send a MOTECH event for each feed content that changed since the last fetch.
```

Comment:
"each feed content" sounds a bit awkward. Should it be each "each feed item" or something like that?